api/src/main/java/com/debugmafia/clueless/model/Weapon.java

```diff
@@ -0,0 +1,21 @@
+package com.debugmafia.clueless.model;
+
+import java.util.UUID;
+
+public class Weapon {
+  private String weaponName;
+  private UUID uuid;
+
+  public Weapon(String name) {
+    this.weaponName = name;
+    uuid = UUID.randomUUID();
+  }
+
+  public String getWeaponname() {
+    return this.weaponName;
+  }
+
+  public UUID getUuid() {
+    return this.uuid;
+  }
+}
```

Comment:
Add the `@Override` for the `equals()` method.

Reply:
I have a rough understanding of what needs to be done there, but I think you should perform the implementation.

Reply:
Will do